pkg/install/generate.go

```diff
@@ -291,11 +291,45 @@ func InstallerPodSpec(
 		},
 	}
 
+	var hostNetwork bool
+	if cd.Spec.Platform.BareMetal != nil {
+		containers = append(containers,
+			corev1.Container{
+				Name: "libvirt",
+				// TODO: need an official image for this
+				Image:           "docker.io/djzager/libvirt:netstart",
+				ImagePullPolicy: corev1.PullAlways,
```

Comment:
do we always want 'PullAlways'? or is this temporary?

Reply:
Temporary, not sure what the longterm image we use here will be, although I think we usually use PullAlways don't we?

Reply:
i was just thinking if whatever container repo that houses these images becomes inaccessible, does that mean that installations grind to a halt even though the image might be cached?

Reply:
Could be a problem, but across the board as I believe we're using this for almost everything.

Reply:
Seems like we want this for development setups so we always get our latest image, but in prod/stage we use hashes so IfNotPresent would work fine there. We should probably make this configurable somewhere, HiveConfig perhaps.